understanding/meaningful-sequence.html

```diff
@@ -58,109 +56,86 @@
             <nav>
-               <header class="box-h ">Page Contents</header>
+               <header class="box-h ">このページの内容</header>
                <div class="box-i">
                   <ul>
-                     <li><a href="#brief">In Brief</a></li>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.3.2:</span>Meaningful Sequence (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書
+                  				達成基準 1.3.2:</span>意味のあるシーケンス (レベル A)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>When the sequence in which content is presented affects its meaning, a <a href="#dfn-correct-reading-sequence">correct reading sequence</a> can be <a href="#dfn-programmatically-determined">programmatically determined</a>.
+                  <p>コンテンツが提示されている順序が意味に影響を及ぼす場合には、<a href="#dfn-correct-reading-sequence">正しく読むシーケンス</a>は<a href="#dfn-programmatically-determined">プログラムによる解釈</a>が可能である。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>The order of content can be understood by more people.</dd>
+                  <dt>目標</dt>
+                  <dd>コンテンツの順序を、より多くの人が理解できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Use code to preserve meaningful content order.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>コードを用いて、意味をなすコンテンツ順序を維持する。</dd>
```

Comment:
コードを追加して順序を維持するのではなく、順序が破綻しないようなやり方でコードを用いる、というニュアンスを出したいです。難しいですが例えばこうですかね?

```suggestion
                  <dd>意味をなすコンテンツ順序を保つように、コードを用いる</dd>
```